src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/JavaScriptEncoder.cs

```diff
@@ -23,6 +23,29 @@ public static JavaScriptEncoder Default
             get { return DefaultJavaScriptEncoder.Singleton; }
         }
 
+        /// <summary>
+        /// Returns a built-in instance of <see cref="JavaScriptEncoder"/> that is less strict about what gets encoded.
+        /// </summary>
+        /// <remarks>
```

Comment:
@mairaw, any feedback on this remarks section of the API? Particularly around wording and how some of the special characters are written (like <, >, &).